lms/views/admin/application_instance.py

```diff
@@ -252,37 +252,54 @@ def show_instance(self):
         return {"instance": ai}
 
     @view_config(
-        route_name="admin.instance.id",
-        request_method="POST",
-        require_csrf=True,
-    )
-    @view_config(
-        route_name="admin.instance.consumer_key",
+        route_name="admin.instance.move_org",
         request_method="POST",
         require_csrf=True,
     )
-    def update_instance(self):
+    def move_application_instance_org(self):
         ai = self._get_ai_or_404(**self.request.matchdict)
 
         try:
             self.application_instance_service.update_application_instance(
                 ai,
-                lms_url=self.request.params.get("lms_url", "").strip(),
```

Comment:
Diff it's a bit strange here.

- No more try/except on `update_instance`
- New view for move_application_instance_org, with the try.

Reply:
It's a bit weird for the service to be raising ValidationErrors which have specific params specified in them. Seems like a mix of view and service concerns.